open-hackathon-server/src/hackathon/views/schema/AdminHostserverResource

```diff
@@ -0,0 +1,143 @@
+            "pattern": "^[0-9]+$",
+            "required": true
+          },
+          "container_count": {
+            "description": "the container count of vm/host_server",
+            "pattern": "^[0-9]+$",
+            "required": true
+          },
+          "container_max_count": {
+            "description": "the max container count of vm/host_server",
+            "pattern": "^[0-9]+$",
+            "required": true
+          }
+        }
+      }
+    },
+
+    "put": {
+      "input": {
+        "title": "request to update the information of a host server",
+        "description": "args to update the information of a host server",
+        "type": "object",
+        "properties": {
+          "id": {
+            "description": "id of the host_server",
+            "pattern": "^[0-9]+$",
+            "required": true
+          },
+          "hackathon_id": {
+            "description": "id of the hackathon",
+            "pattern": "^[0-9]+$",
+            "required": true
+          },
+          "vm_name": {
+            "description": "the name of vm/host_server",
+            "pattern": "^[0-9]+$",
+            "required": true
+          },
+          "public_dns": {
+            "description": "the public_dns of vm/host_server",
+            "pattern": "^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$",
+            "required": true
+          },
+          "public_ip": {
+            "description": "the public_ip of vm/host_server",
+            "pattern": "^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$",
+            "required": true
+          },
+          "public_docker_api_port": {
+            "description": "the public_docker_api_port of vm/host_server",
+            "pattern": "^[0-9]+$",
+            "required": true
+          },
+          "private_ip": {
+            "description": "the private ip of vm/host_server",
+            "pattern": "^((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)$",
+            "required": true
+          },
+          "private_docker_api_port": {
+            "description": "the private_docker_api_port of vm/host_server",
+            "pattern": "^[0-9]+$",
+            "required": true
+          },
+          "container_count": {
+            "description": "the container count of vm/host_server",
+            "pattern": "^[0-9]+$",
+            "required": true
+          },
+          "container_max_count": {
+            "description": "the max container count of vm/host_server",
+            "pattern": "^[0-9]+$",
+            "required": true
+          }
+        }
+      }
+    },
+
+    "delete": {
+      "input": {
+        "title": "request to delete a host server",
+        "description": "args to delete a host server for a hackathon",
+        "type": "object",
+        "properties": {
```

Comment:
Why there is no properties? Is `id` of host server required? If not, please remove the whole `delete` section